Color selectable hoodies, actually fixes black jumpskirt icon

babydoll opened the pull request:
Adds color selectable hoodies to the suits loadout menu.
Actually fixes the black jumpskirt icon. #14963
Adds a changelog for the crew manifest change #14988

Credit to Zuhayr, I believe, for the original icons.

babydoll committed:
Adds color changeable hoodies to loadout menu
Actually fixes black jumpskirt icon
Changelog for crew manifest

PsiOmegaDelta commented:
Are the black/grey hoodie icons sufficiently different from the added one to warrant still being allowed to exist?

sabiram commented:
I'll get some screenshots here shortly.

babydoll commented:
I tried to approximate the color of the preset colors.
RGB grey - grey hoodie
![ss_1481458580](https://cloud.githubusercontent.com/assets/8846073/21080004/dd51ce20-bf71-11e6-94a8-75d9b8c01f9d.png)

Pure white RGB - RGB black - black hoodie
![ss_1481458633](https://cloud.githubusercontent.com/assets/8846073/21080005/e0668d26-bf71-11e6-8736-edd2dd3e19a6.png)

Few different colors
![ss_1481459031](https://cloud.githubusercontent.com/assets/8846073/21080045/c9e18ba4-bf72-11e6-945d-fe77d0397e8b.png)

Looking at them now, I don't have a strong preference either way. Once I had the models set up, I did have to check which ones were wearing which while writing this comment.

babydoll committed:
Loads of adjustments

Removes grey hoodie and black hoodie icons
Retains black hoodie as a recolored subtype for compatibility
Enforces loadout menu naming scheme standard
